src/components/SelectionList/BaseSelectionList.js

```diff
@@ -311,7 +312,7 @@ function BaseSelectionList({
                 isFocused={isItemFocused}
                 isDisabled={isDisabled}
                 isHide={!maxToRenderPerBatch}
-                showTooltip={showTooltip}
+                showTooltip={showTooltip && shouldShowTooltip}
```

Comment:
The variable names are getting a bit confusing here, I think it's more readable if we move this conditional to the const above:

```
const showTooltip = shouldShowTooltip && normalizedIndex < 10;
```

Also very minor but maybe renaming `shouldShowTooltips` (as a plural) works better to indicate it's for the whole list, rather than one tooltip!